spec/smoke_test_spec.lua

```diff
@@ -1,10 +1,15 @@
 package.path = "target/?.lua;" .. package.path
 require("busted.runner")()
+local Reader = require("spec.pom.Reader")
 
-local pom = require("spec.pom.reader")
+local function get_project_base_path()
+    return debug.getinfo(1,"S").source:sub(2):gsub("[^/]*$", "") .. ".."
+end
```

Comment:
This is duplicated three times.
Maybe add a method `read_default_pom` to the Reader?